base/applications/taskmgr/perfdata.c

```diff
@@ -389,6 +390,10 @@ void PerfDataRefresh(void)
                     pPerfData[Idx].GDIObjectCount = GetGuiResources(hProcess, GR_GDIOBJECTS);
                 }
 
+                if (IsWow64Process(hProcess, &bIsWow64) && bIsWow64) {
```

Comment:
```suggestion
                if (IsWow64Process(hProcess, &bIsWow64) && bIsWow64)
                {
```

Reply:
Could you also move this up to where the contents of the ImageName buffer gets initialized?